src/MovieCard/MovieCard.js

```diff
@@ -12,12 +12,18 @@ const MovieCard = ({ userID, title, averageRating, poster, id, userRatings }) =>
 	
 	return (
 		<section className='movie-card' id={id}>
-			<h3>{title}</h3>
-			<p className={averageRating > 5 ? 'green-rating' : 'red-rating'}>
-				{foundRating ? 
-					`You Rated ${foundRating}` : 
-					`Average Rating ${Math.floor(averageRating)}`}/10
-			</p>
+			<section className='title-section'>
+				<h3>{title}</h3>
+			</section>
```

Comment:
you can add a className to the h3 tag, if that's easier here instead of adding another section tag to wrap it